sources/rest_api/client.py

```diff
@@ -32,12 +30,20 @@ def __init__(
         auth: Optional[AuthBase] = None,
         paginator: Optional[BasePaginator] = None,
         session: BaseSession = None
+        request_client: Client = None,
+        ignore_http_status_codes: Optional[Sequence[int]] = [],
```

Comment:
Totally missed that but it's a [dangerous default value](https://pylint.pycqa.org/en/latest/user_guide/messages/warning/dangerous-default-value.html)

Reply:
Uh, thank you, I didn't know that! I can fix it so that we pacify the linter.